sunpy/net/scraper_utils.py

```diff
@@ -0,0 +1,148 @@
+import calendar
+from datetime import datetime, timedelta
+
+from dateutil.relativedelta import relativedelta
+
+from sunpy.extern.parse import parse
+from sunpy.time import TimeRange
+
+TIME_QUANTITIES = {'day': timedelta(days=1),
+                   'hour': timedelta(hours=1),
+                   'minute': timedelta(minutes=1),
+                   'second': timedelta(seconds=1),
+                   'millisecond': timedelta(milliseconds=1)}
+
```

Comment:
You need to add an __all__ here. That should fix the doc build.

Reply:
I don't really follow how I should add an **all** here, can you please elaborate?

Reply:
In this format: `__all__ = ["function_name_1", "function_name_2", "funtion_name_3"]`

Reply:
ahh... got it, i was thinking that the issue must be something along those lines but couldn't put a finger on it, thanks!